### PR TITLE
Move file config/global/configuration.yml -> config/global.yml

### PR DESCRIPTION
**Pull request by pankajkoti**
Based on previous discussions and observations from users, it seems that having a global directory creates confusion because global itself is not any environment and the directories within config directory are meant to be environments. So, we are no longer creating the global directory as part of the initialised project and moving the global configuration to top level global.yml file in the config directory.

related: #1230

### CODE REVIEW

File: sql-cli/tests/config/test/configuration.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ connections:
     schema: pagila
     login: postgres
     password: $POSTGRES_PASSWORD
-    port: $POSTGRES_PORT
+    port: 5432
```
Comment (tatiana) [inline sql-cli/tests/config/test/configuration.yml:25]: Is this change intended, @pankajkoti ?

**Review by tatiana — APPROVED** [review]

File: sql-cli/tests/config/test/configuration.yml (hunk 1 above)
Comment (pankajkoti) [reply sql-cli/tests/config/test/configuration.yml:25]: yes, the change is intentional. A validate test was failing on my local saying not a valid integer.

Comment (feluelle) [reply sql-cli/tests/config/test/configuration.yml:25]: @pankajkoti this seems to be an issue to your local environment/setup. I am not having the issue and using `POSTGRES_PORT=5433` in my `.env` file. Now I am unable to use the same db. I would be in favor of reverting this change, if that's okay for you both? @tatiana

File: sql-cli/sql_cli/project.py [hunk 2]
```diff
@@ -82,7 +82,7 @@ def _initialise_global_config(self) -> None:
         Initialises global config file that includes configuration to be shared across environments including the
         airflow config.
         """
-        config = Config(environment=GLOBAL_CONFIG, project_dir=self.directory)
+        config = Config(environment=DEFAULT_ENVIRONMENT, project_dir=self.directory)
```
Comment (feluelle) [inline sql-cli/sql_cli/project.py:85]: @pankajkoti why is `environment=DEFAULT_ENVIRONMENT` for the "global" configuration?

File: sql-cli/tests/config/test/configuration.yml (hunk 1 above)
Comment (pankajkoti) [reply sql-cli/tests/config/test/configuration.yml:25]: It is an issue if the environment variable in not set

File: sql-cli/sql_cli/project.py (hunk 2 above)
Comment (pankajkoti) [reply sql-cli/sql_cli/project.py:85]: The Config instance has environment as a required key. We can set it to anything as it does not have any effect for the global config keys.

Comment (feluelle) [reply sql-cli/sql_cli/project.py:85]: > We can set it to anything as it does not have any effect

^this is huge code smell in my opinion.

File: sql-cli/tests/config/test/configuration.yml (hunk 1 above)
Comment (pankajkoti) [reply sql-cli/tests/config/test/configuration.yml:25]: okay, will add back the env var.
